java/dictionary-renderer-graphics-overlay/README.md

```diff
@@ -19,17 +19,17 @@ This sample loads a number of point, line, and polygon military elements from an
 1. Download the data from the table below.
```

Comment:
The readme has been redone as part of the big overhaul. Might be a good point in time to update this one to match the design readme. It has a use-case too!